lms/djangoapps/courseware/courses.py

```diff
@@ -224,14 +224,15 @@ def get_course_about_section(course, section_key):
                 except Exception:  # pylint: disable=broad-except
                     html = render_to_string('courseware/error-message.html', None)
                     log.exception(
-                        u"Error rendering course={course}, section_key={section_key}".format(
-                            course=course, section_key=section_key
-                        ))
+                        u"Error rendering course=%s, section_key=%s",
```

Comment:
Why is this change necessary? I find it harder to read with ``%s``. Does PyLint not like this idiom?